### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by abenaTenkorang**
### We are following this template [Template1](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and made the bellow changes

> In this pull request we added HTML, CSS, and Javascript linter.

> In mobile, we brainstormed and implemented the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.

- When the user clicks (or taps) the close (X) button, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -48,11 +48,62 @@ body {
...
+  /* background: white; */
+  cursor: pointer;
+  color: #6070ff;
+}
+
+.close-icon {
+  display: none;
+  background: #6070ff;
+  color: #fff;
+}
```
Comment (Gopxfs) [inline style.css]: To stick with the original design, please make sure that your closing icon looks like the one displayed in Figma 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Kindly notice that your button has a darker square around it 😄.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: style.css [hunk 2]
```diff
@@ -48,11 +48,62 @@ body {
...
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  opacity: 0.9;
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+  padding-left: 2rem;
+  gap: 12px;
+}
```
Comment (Gopxfs) [inline style.css]: Please notice that, in the original design, the background is blurred when the mobile menu appears 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: style.css [hunk 3]
```diff
@@ -48,11 +48,62 @@ body {
...
+
+.showMenu {
+  transform: translateY(0);
+}
```
Comment (Gopxfs) [inline style.css]: [OPTIONAL] You did a great job with your mobile menu, well done 😄! I liked the animation effect as well. Please take into account that most of the mobile menus stick to the user's screen. This means that, when scrolling down, the menu is still being displayed at the top of the page. Currently, your menu is static on the beginning of your page, causing this effect when scrolling down:
image

So, to improve the UX/UI of your webpage, kindly consider this advice 😄.

> You could do it by giving a position: fixed to your menu 😉.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @abenaTenkorang @Franky237 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ The menu is working as intended.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (abenaTenkorang) [reply style.css]: Very educational👌. Thank you so much

File: style.css (hunk 2 above)
Comment (abenaTenkorang) [reply style.css]: Issue resolved. Opacity has been increased to blur the background since background could not be blurred directly

**Comment by abenaTenkorang** [comment]
All issues have been resolved. Thank you.

File: style.css [hunk 4]
```diff
@@ -67,12 +67,11 @@ body {
 
 .close-icon {
   display: none;
-  background: #6070ff;
   color: #fff;
 }
 
 .menu {
```
Comment (ridwanediallo) [inline style.css:71]: - [ ] You did on implementing the changes requested by the previous reviewer 👍️ but, still, the background is not blurred. 
Adding this  👉️`backdrop-filter: blur(5px);`  to your `menu` class should do the work.

**Review by ridwanediallo — CHANGES_REQUESTED** [review]

File: style.css (hunk 4 above)
Comment (abenaTenkorang) [reply style.css:71]: @ridwanediallo I added and still does not look blurred.
Please can you send me your mail so I send for you to take a look

Comment (ridwanediallo) [reply style.css:71]: Add this  `    background-color: #396df2e5;
    backdrop-filter: blur(4px);`  to your Showmenu class.

**Comment by abenaTenkorang** [comment]
All requested changes have been made.

**Comment by abenaTenkorang** [comment]
All requested updates have been made.

File: style.css (hunk 4 above)
Comment (abenaTenkorang) [reply style.css:71]: Thank you very much, it worked

**Review by Olamarx — APPROVED** [review]
Hi @abenaTenkorang, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✔️ 
- Git and GitHub flow 👍🏼 
- Organized code ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — COMMENTED** [review]
Hi Abi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**